server/restapi/models.py

```diff
@@ -0,0 +1,9 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+
+
+class Datauser(models.Model):
+    global_id = models.UUIDField(verbose_name='id', editable=False, default=uuid.uuid4)
+    name_user = models.CharField(verbose_name='name', max_length=20, unique=True)
+    create_date = models.DateTimeField(verbose_name='data', editable=False, default=timezone.now())
```

Comment:
При запуске выдает предупреждение

```
System check identified some issues:

WARNINGS:
restapi.Datauser.create_date: (fields.W161) Fixed default value provided.
	HINT: It seems you set a fixed date / time / datetime value as default for this field. This may not be what you want. If you want to have the current date as default, use `django.utils.timezone.now`
```

Оно скорее всего не просто так и связано с с тем, что ты записываешь в defaults значение, а должен передать фунцию (callable). И это приводит к тому, что create_date всегда один и тот же.